Package.swift

```diff
@@ -72,7 +72,7 @@ let package = Package(
     targets: [
         .target(
             name: "TuistGraph",
-            dependencies: [swiftToolsSupportDependency]
+            dependencies: [swiftToolsSupportDependency, "ProjectDescription"]
```

Comment:
Why do you need this dependency? I don't think the `TuistGraph` should depend on the `ProjectDescription`